### ISSUE TITLE
Extended Model for Resistivity in Pygimli

### ISSUE THREAD

**Issue by tonapawilliam** [gimli-org/gimli#684]
Excuse Me Sir,
Is there a function in Pygimli ERT to output the model inversion results in the form of an extended model like in res2dinv, so that the output data can be shaped like an inverted trapezoid or triangle? If so, can the data be exported in accordance with the shape of the extended inversion model only? Thank you

**Comment by halbmy** [comment]
In res2dinv, the model is trapezoidal-shaped
![image](https://github.com/gimli-org/gimli/assets/7136401/82e5c029-bcc7-4b84-b17a-d14288540deb)
which can be unfavorable for certain conductivity distributions and with topography in general.
Of course you can easily create such a trapezoidal model shape in pyGIMLi as well, maybe it could be beneficial to add such an option to `meshtools.createParaDomainPLC`.

However, you can blend out poorly covered cells by using
```
mgr.showResult(coverage=mgr.standardizedCoverage(threshold))
```
e.g. for the gallery (2d flat topo example) case, `mgr.showResult(coverage=mgr.standardizedCoverage(-1.3))` leads to
![image](https://github.com/gimli-org/gimli/assets/7136401/a9eb17bd-cf5a-423a-b270-f4b7ef71ef06)

**Comment by halbmy** [comment]
Just for fun, I just added (8050bef3) a keyword argument `trapRatio` to `createParaMeshPLC` that might help to reduce the number unknowns and also the ill-posedness of the inverse problem. The example
```
mgr = ert.Manager(data)
mgr.invert(trapRatio=0.3, paraDepth=12)
mgr.showResult(coverage=1)
```
yields
![image](https://github.com/gimli-org/gimli/assets/7136401/06a68108-ea09-4a6c-984c-9ad372547d7d)

**Comment by tonapawilliam** [comment]
Thank You Sir. When I export this data into x y res, can this trapezoidal model also come out so that when plotted based on x y, it will also form a trapezoid? I find it quite problematic when I take out all the data, including the extrapolated ones that cause my inversion data to reach more than 7000.

**Comment by tonapawilliam** [comment]
And I want to know, in my case, I want a translucent geological model that can be attached to the inversion results. Do I use createMeshPLC or is there another feature for that? Maybe the following picture can clarify what I mean
![image](https://github.com/gimli-org/gimli/assets/126912836/50e82a84-4a61-4a5f-853e-4293c6853406)

**Comment by halbmy** [comment]
For using the `trapRatio` argument, you will have to checkout pyGIMLi by git as described by https://www.pygimli.org/installation.html#staying-up-to-date

**Comment by halbmy** [comment]
Yes, this is finally closed with version 1.5.2. Thank you for your feedback.

**Comment by tonapawilliam** [comment]
Yes, thank you Sir, this is my result 
![Syntetic Vale E05](https://github.com/user-attachments/assets/6b687577-5c51-4095-b7ef-00779f88dddf)
![Vale N05](https://github.com/user-attachments/assets/3c6d0852-0e7d-4b15-81da-bcedbd9d988d)
